addons/website_customer/views/website_customer_templates.xml

```diff
@@ -198,7 +199,8 @@
         </h4>
         <div><a t-attf-href="/partners/#{slug(partner.assigned_partner_id)}"
                 t-field="partner.assigned_partner_id.image_128"
-                t-options='{"widget": "image", "class": "d-block mx-auto"}'
+                style="display: inline-block; width: 128px"
+                t-options='{"widget": "image", "class": "d-inline-block mx-auto o_image_128_max"}'
```

Comment:
`mx-auto` acts like `text-center` for block elements, so maybe we should use that everywhere for consistency instead of `text-center`.
https://getbootstrap.com/docs/4.4/utilities/spacing/#horizontal-centering

Reply:
`mx-auto` conflicts with `display: inline-block!important` from `d-inline-block`.
If we remove it, `mx-auto` causes `display: block`, so it is OK.
I will push this.